test/core/componentGroupTests.ts

```diff
@@ -185,6 +185,48 @@ describe("ComponentGroups", () => {
       assert.strictEqual(cg.height(), SVG_HEIGHT, "occupies all offered height");
       svg.remove();
     });
+
+    it("can move components to other groups after anchoring", () => {
+      var svg = generateSVG();
+
+      var xScale = new Plottable.Scale.Category();
+      xScale.domain(["A", "B", "C"]);
+      var xAxis = new Plottable.Axis.Category(xScale, "bottom");
+      var yScale = new Plottable.Scale.Linear();
+      var yAxis = new Plottable.Axis.Numeric(yScale, "left");
+      var chart = new Plottable.Component.Table([
```

Comment:
You shouldn't have to use a `Table` in this test; simply create two `Group`s in different `<svg>`s and move a `Component` between them after rendering.